app/views/projects/_status_chooser.html.erb

```diff
@@ -18,6 +18,7 @@
     <div class="col-md-3 col-sm-4 col-xs-4 criteria-radio<%= crypto_class %>">
       <div class="status-chooser">
       <%= f.form_group status_symbol do %>
+        <a href="#<%= criterion.to_s %>">
```

Comment:
You should use `link_to`  https://apidock.com/rails/ActionView/Helpers/UrlHelper/link_to

Reply:
Thanks! Haven't done much ERB so tips like this are helpful. Does this look right?

Reply:
That looks better and more idiomatic. I haven't tested it locally but if it behaves as intended, LGTM

Reply:
Thank you!